src/sentry/receivers/onboarding.py

```diff
@@ -143,13 +144,44 @@ def record_first_event(project, event, **kwargs):
         )
         return
 
+    url = None
+
+    # Check for the event url
+    for key, value in event.tags:
+        if key == "url":
+            url = value
+            break
+
+    # Check if an event contains a minified stack trace
+    has_minified_stack_trace = False
+
+    exception_values = get_path(event.data, "exception", "values", filter=True)
+
+    if exception_values:
+        for exception_value in exception_values:
+            if "raw_stacktrace" in exception_value:
+                has_minified_stack_trace = True
+                break
+
+    if has_minified_stack_trace:
+        analytics.record(
+            "first_event_with_minified_stack_trace_for_project.sent",
```

Comment:
@priscilawebdev I don't think you achieve anything with this second event.  This function only fires on the first event, meaning you already are sending this info in the `first_event_for_project.sent` event. If the first event has no source maps but the second does, you're not going to emit a `first_event_with_minified_stack_trace_for_project.sent` which is likely going to be confusing.